extensions/wikia/SitemapPage/SitemapPageModel.class.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+class SitemapPageModel extends WikiaModel {
+
+	const SITEMAP_PAGE = 'Sitemap';
+	const CACHE_VERSION = '1';
+	const CACHE_TTL = 86400;
+	const DEFAULT_LIMIT_PER_PAGE = 100;
+	const MAX_LEVEL = 3;
+	const VERTICAL_UNKNOWN = 'Unknown';
+
+	protected static $verticalNames = [
+		WikiFactoryHub::VERTICAL_ID_OTHER => 'Other',
+		WikiFactoryHub::VERTICAL_ID_TV => 'TV',
+		WikiFactoryHub::VERTICAL_ID_VIDEO_GAMES => 'Games',
+		WikiFactoryHub::VERTICAL_ID_BOOKS => 'Books',
+		WikiFactoryHub::VERTICAL_ID_COMICS => 'Comics',
+		WikiFactoryHub::VERTICAL_ID_LIFESTYLE => 'Lifestyle',
+		WikiFactoryHub::VERTICAL_ID_MUSIC => 'Music',
+		WikiFactoryHub::VERTICAL_ID_MOVIES => 'Movies',
+	];
+
+	/**
+	 * Get limit per page
+	 * @param int $level
+	 * @return int
+	 */
+	public function getLimitPerPage( $level = 0 ) {
+		$base = $this->getLimitPerPageBase();
+		return ( $level == 1 ) ? pow( $base, self::MAX_LEVEL - 1 ) : $base;
```

Comment:
What makes `$level == 1` have such a big difference in limit per page? Documentation could be helpful.